src/modules/NtpClient.rb

```diff
@@ -372,8 +372,9 @@ def Import(settings)
         Yast::Report.Error(
           format(
             # TRANSLATORS: error report. %s stands unsuported keys.
-            _("Ignoring the NTP configuration. The profile format has changed in an " \
-              "incompatible way. These keys are no longer supported: '%s'."),
+            _("Ignoring the NTP configuration. The profile format has changed in an\n" \
```

Comment:
maybe just mention it in comment for translators, that they also have to fit into such dimenses and if needed, move `\n`